frontend/translations/es.po

```diff
@@ -4753,6 +4752,18 @@ msgstr "**$%s**/mes"
 msgid "subscription.settings.management.dialog.title"
 msgstr "Aplica %s a tus equipos"
 
+msgid "subscription.settings.management.dialog.step-2.title"
+msgstr "Ayúdanos a crecer y facilita tu periodo de prueba"
+
+msgid "subscription.settings.management.dialog.step-2.description"
```

Comment:
This translation keys does not match the translations guideline: https://docs.google.com/document/d/1ZB5n8XyFjA_XSUYF5R-2sHEsTvsozXRDe4T-QABfTDA/edit?usp=sharing

My suggestion for this key is: `subscription.management-dialog.step-2-description`